spec/lib/formatter/csv/annotation_for_csv_spec.rb

```diff
@@ -138,67 +138,95 @@
       end
 
       context "with a combo task" do
-        let(:combo_workflow) { build(:workflow, :complex_task) }
```

Comment:
Combo and complex are separate workflow/workflow content traits in the factories, but this seems to be describing a "combo" but using the "complex" factory. They may be functionally interchangeable, but this might be confusing.